src/dev-app/tabs/tabs-demo.html

```diff
@@ -1,5 +1,7 @@
 <h3>Tab group alignment</h3>
 <tab-group-align-example></tab-group-align-example>
+<h3>Tab group dynamic alignment</h3>
+<tab-group-dynamic-align-example></tab-group-dynamic-align-example>
```

Comment:
I don't think we need two examples with a static and dynamic alignment. Maybe we can update the existing `tab-group-align` example to be dynamic instead?